<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelFace">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>261</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string>labelFace</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLegCam">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>541</height>
    </rect>
   </property>
   <property name="text">
    <string>labelCam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCamOn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cam on</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>94</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="plotDiff">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>221</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plotPerson">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>350</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>190</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTest">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>390</y>
     <width>94</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
